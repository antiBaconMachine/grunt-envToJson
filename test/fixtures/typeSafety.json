{
    "foo": true,
    "bar": 12
}