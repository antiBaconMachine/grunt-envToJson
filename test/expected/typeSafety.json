{
    "foo": false,
    "bar": 21
}